File: Iniciante/1080 - Maior e Posição/TypeScript/1080.ts
import * as fs from "fs";

const QUANTITY_OF_VALUES: number = 100;

function main(): void {
    const input: string[] = fs.readFileSync(0, 'utf8').trim().split('\n');
    
    let higherValue: number = Number.NEGATIVE_INFINITY;;
    let position: number = 0;

    for (let i = 0; i < QUANTITY_OF_VALUES; i++) {
        const value: number = parseInt(input[i]);
        if (value > higherValue) {
            higherValue = value;
            position = i + 1;
        }
    }

    console.log(higherValue);
    console.log(position);
}

main();

File: Iniciante/1080 - Maior e Posição/Python/1080.py
QUANTITY_OF_VALUES = 100

def main():
    higher_value = float('-inf')
    position = 0

    for i in range(QUANTITY_OF_VALUES):
        value = int(input())
        if value > higher_value:
            higher_value = value
            position = i + 1

    print(higher_value)
    print(position)

if __name__ == "__main__":
    main()
